docs/src/cli/delegate-stake.md

```diff
@@ -0,0 +1,136 @@
+
+Create a stake account with the `solana create-stake-account` command:
+
+```bash
+solana create-stake-account --from=<KEYPAIR> <ACCOUNT_KEYPAIR> <AMOUNT>
+```
+
+`<AMOUNT>` tokens are transferred from the account at `<KEYPAIR>` to a new
+stake account at the public key of `<ACCOUNT_KEYPAIR>`.  Record the account
+public key, as it will be used later to perform actions on the stake account.
+The keypair file can then be discarded. To authorize additional actions, you
+will use `<KEYPAIR>`, not `<ACCOUNT_KEYPAIR>`.
+
+### Advanced: Derive Stake Account Addresses
+
+When we delegate stake, we delegate all tokens in the stake account to a single
+validator. To delegate to multiple validators we will need multiple stake
+accounts. Creating a new keypair for each account and managing those addresses
+can be cumbersome. Fortunately, we can derive stake addresses using the `--seed`
+option:
+
+```bash
+solana create-stake-account --from=<KEYPAIR> <KEYPAIR> --seed=<STRING> <AMOUNT>
+```
+
+`<STRING>` is an arbitrary string up to 32 characters, but will typically be
+a number corresponding to which derived account this is. The first account
+might be "0", then "1", and so on. `<KEYPAIR>` is the same for both the source
+keypair and account keypair, because the second instance is used only as a base
+address.  The command derives a new address from the base address and seed
+string. To see what stake address the command will derive, use
+`solana create-address-with-seed`:
+
+```bash
+solana create-address-with-seed --from=<PUBKEY> <SEED_STRING> STAKE
+```
+
+It will output a derived address, which can be used for the
+`<ACCOUNT_PUBKEY>` argument in staking operations.
+
+## Set Stake and Withdraw Authorities
+
+Staking commands look to keypairs to authorize certain stake account
+operations. They use the stake authority to authorize stake delegation,
+deactivating stake, splitting stake, and setting a new stake authority.  They
+use the withdraw authority to authorize withdrawing stake, and when setting either
+a new stake or withdraw authority.
+
+Stake and withdraw authorities can be set when creating account via the
+`--stake-authority` and `--withdraw-authority` options, or afterward with
+the `solana stake-authorize` command. For example, to set a new authorized
+staker, run:
+
+```bash
+solana stake-authorize <ACCOUNT_KEYPAIR> --stake-authority=<KEYPAIR> --new-stake-authority=<PUBKEY>
+```
+
+This will use the existing stake authority `<KEYPAIR>` to authorize a new
+stake authority `<PUBKEY>` on the stake account `<ACCOUNT_KEYPAIR>`.
+
+## Delegate Stake
+
+To delegate stake, run:
+
+```bash
+solana delegate-stake --stake-authority=<KEYPAIR> <ACCOUNT_PUBKEY> <VOTE_PUBKEY>
+```
+
+`<KEYPAIR>` authorizes the operation on the account with address
+`<ACCOUNT_PUBKEY>`. The stake is delegated to the vote account with
+public key `<VOTE_PUBKEY>`.
+
+## Deactivate Stake
+
+Once delegated, you can deactivate stake with the `solana deactivate-stake`
```

Comment:
s/Once delegated/To stop delegating your stake once it has been delegated/

Just to make it clear that `dectivate` is the opposite of `delegate`, which is not obvious, particularly next to `withdraw` which sounds like similar behavior.  Can we just re-name it `undelegate`?